### PR TITLE
Support ARMv6 devices.

### PR DESCRIPTION
**Pull request by talregev**
APK 7.4.53 do not support armv6 devices. 
I create a PR, compile and run 3 projects on my 2 armv6 devices.
My device: samsung galaxy gio GT-S5660, samsung galaxy pop plus GT-S5570i
Projects: einstein@home, yoyo@home, moo! wrapper
Also I uploaded the compiled binaries as @AenBleidd requested. 

armv6 finished tasks at the 3 projects: 
https://einsteinathome.org/host/12873266
https://moowrap.net/show_host_detail.php?hostid=1309533
https://www.rechenkraft.net/yoyo/show_host_detail.php?hostid=500196

Some screenshots:
IMG_20210219_223401 1
IMG_20210219_215331_1 1
IMG_20210219_222219_1

### CODE REVIEW

File: android/BOINC/app/src/main/assets/all_projects_list.xml [hunk 1]
```diff
@@ -0,0 +1,933 @@
+<?xml version="1.0" encoding="ISO-8859-1" ?>
```
Comment (AenBleidd) [inline android/BOINC/app/src/main/assets/all_projects_list.xml:1]: This file should be removed completely from here

File: android/BOINC/app/src/main/assets/ca-bundle.crt [hunk 2]
```diff
@@ -0,0 +1,3410 @@
+##
```
Comment (AenBleidd) [inline android/BOINC/app/src/main/assets/ca-bundle.crt:1]: This file should be removed completely from this PR

File: client/hostinfo_unix.cpp [hunk 3]
```diff
@@ -109,6 +109,10 @@
 #include "win/opt_x86.h"
 #endif
 
+#ifdef ARMV6
+    #include <sys/wait.h>
```
Comment (AenBleidd) [inline client/hostinfo_unix.cpp:113]: remove indent

**Review by AenBleidd — CHANGES_REQUESTED** [review]
Next files should be removed too:
- android/BOINC/app/src/main/assets/arm64-v8a/boinc
- android/BOINC/app/src/main/assets/armeabi-v7a/boinc
- android/BOINC/app/src/main/assets/armeabi/boinc
- android/BOINC/app/src/main/assets/x86/boinc
- android/BOINC/app/src/main/assets/x86_64/boinc

**Comment by talregev** [comment]
I will push force without the boinc binaries and xml and certificates, then continue to other fixes.

File: client/hostinfo_unix.cpp (hunk 3 above)
Comment (talregev) [reply client/hostinfo_unix.cpp]: I remove it

File: android/BOINC/app/src/main/assets/ca-bundle.crt (hunk 2 above)
Comment (talregev) [reply android/BOINC/app/src/main/assets/ca-bundle.crt]: I remove this file

File: android/BOINC/app/src/main/assets/all_projects_list.xml (hunk 1 above)
Comment (talregev) [reply android/BOINC/app/src/main/assets/all_projects_list.xml]: I remove this file

**Comment by talregev** [comment]
> Next files should be removed too:
> 
> * android/BOINC/app/src/main/assets/arm64-v8a/boinc
> * android/BOINC/app/src/main/assets/armeabi-v7a/boinc
> * android/BOINC/app/src/main/assets/armeabi/boinc
> * android/BOINC/app/src/main/assets/x86/boinc
> * android/BOINC/app/src/main/assets/x86_64/boinc

I remove these files.

**Review by AenBleidd — APPROVED** [review]

**Pull request merged**